raiden/tests/utils/transfer.py

```diff
@@ -115,6 +115,24 @@ def get_channelstate(
     return channel_state
 
 
+def create_route_state_for_route(
+    apps: List[RaidenService], token_address: TokenAddress, fee_estimate: FeeAmount
+) -> RouteState:
+    assert len(apps) > 1, "Need at least two nodes for a route"
+    route = [app.address for app in apps]
```

Comment:
Would it make sense for this helper to check that there are actually channels between the nodes in the given order?

Reply:
That makes sense. I'll add that in a different PR.